### PR TITLE
Updated default "pager" behavior

### PR DESCRIPTION
**Pull request by codegamc**
This is to address [this issue](https://github.com/maxbla/last-opened-log/issues/1).

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -16,11 +16,11 @@ fn main() -> Result<(), Box<dyn Error>> {
                 .short("p")
                 .long("pager")
                 .value_name("PAGER")
-                .help("Select which pager program to use ")
+                .help("Prints name of last edited file, or use -p to Select which pager program to use.")
                 .takes_value(true),
         )
         .get_matches();
-    let pager = matches.value_of("pager").unwrap_or("less");
+    let pager = matches.value_of("pager").unwrap_or("echo");
```
Comment (maxbla) [inline src/main.rs]: what if the user's pager is called echo?

Comment (codegamc) [reply src/main.rs]: Then they won't notice 😁.

But jokes aside, there are a few strings that can be used here as a flag to indicate that the filename should be printed out (eg ";" would not be a valid command line utility). Alternatively, a larger structural change is necessary to provide the same behavior without overriding that string.

Comment (maxbla) [reply src/main.rs]: `;` actually is [valid](https://unix.stackexchange.com/questions/230291/what-characters-are-valid-to-use-in-filenames), although unwise. I would much prefer the more substantial change.

File: src/main.rs [hunk 2]
```diff
@@ -16,11 +16,11 @@ fn main() -> Result<(), Box<dyn Error>> {
                 .short("p")
                 .long("pager")
                 .value_name("PAGER")
-                .help("Select which pager program to use ")
+                .help("Prints name of last edited file, or use -p to Select which pager program to use.")
```
Comment (maxbla) [inline src/main.rs]: capitalization

File: src/main.rs (hunk 1 above)
Comment (codegamc) [reply src/main.rs]: Updated to use a boolean flag.

File: src/main.rs (hunk 2 above)
Comment (codegamc) [reply src/main.rs]: Updated

**Comment by maxbla** [comment]
replaced by https://github.com/maxbla/last-opened-log/pull/3

**Pull request closed without merging**